entrypoint.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/sh -l
 
-cp /evaluator.js /github/workspace
+git clone https://github.com/$GITHUB_REPOSITORY-tests.git /project-tests
+rm -rf /project-tests/.git
+cp -r /project-tests/* .
 npm install
 npm install -g react-scripts
 CI=true react-scripts test --json --outputFile=evaluation.json
-node evaluator.js evaluation.json requirements_mapping.json result.json
+node /evaluator.js evaluation.json requirements_mapping.json result.json
```

Comment:
Why adding this slash?

Reply:
After I move the `evaluator.js` from `/` to `/github/workspace`, but it is not necessary. So, I kept `evaluator.js` in `/` and use `/evaluator.js` here.